docs/start-here.md

```diff
@@ -20,8 +20,8 @@ What's the problem we're trying to solve?
 
 This is the definition I typically give:
 
-> Given an information need expressed as a query _q_, the text retrieval task is to return a ranked list of _k_ texts {_d<sub>1</sub>_, _d<sub>2</sub>_ ... _d<sub>k</sub>_} from an arbitrarily large but finite collection
-of texts _C_ = {_d<sub>i</sub>_} that maximizes a metric of interest, for example, nDCG, AP, etc.
+> Given an information need expressed as a query _q_, the text retrieval task is to return a ranked list of _k_ texts {_d`<sub>`1`</sub>`_, _d`<sub>`2`</sub>`_ ... _d`<sub>`k`</sub>`_} from an arbitrarily large but finite collection
```

Comment:
nah, I think I like original better...

Reply:
Yeah same here, I didn't mean to change that. My editor did that automatically for some reason. Will fix.